package.json

```diff
@@ -107,7 +107,7 @@
 		"jsdoc": "=3.4.3",
 		"jsdox": "=0.4.10",
 		"lint-staged": "=6.1.0",
-		"lisk-js": "1.0.0-beta.2",
+		"lisk-elements": "beta",
```

Comment:
This should be pinned to `1.0.0-beta.4` to avoid unexpected breakages if we release another beta version.